### PR TITLE
Refine  bernoulli and unsqueeze op

### PR DESCRIPTION
**Pull request by zhiqiu**
### PR types

New features
### PR changes

OPs
### Describe

- add check for bernoulli cuda kernel

- register bool for unsqueeze kernel

- refine doc of paddle.stack

### CODE REVIEW

**Review by jzhang533 — DISMISSED** [review]
lgtm

File: paddle/fluid/operators/bernoulli_op.cu [hunk 1]
```diff
@@ -31,6 +31,10 @@ struct BernoulliCudaFunctor {
   __host__ __device__ BernoulliCudaFunctor(int seed) : seed_(seed) {}
 
   __host__ __device__ T operator()(const unsigned int n, const T p) const {
+    // NOTE(zhiqiu): currently, PADDLE_ENFORCE in cuda kernel may print several
+    // lines of error messages if, and it should be refined.
+    PADDLE_ENFORCE(p >= 0.0 && p <= 1.0,
+                   "The probability should >=0 and <= 1, but got %f", p);
```
Comment (chenwhql) [inline paddle/fluid/operators/bernoulli_op.cu]: 这里是`should >= 0`，下面是`should be <= 1`，要不要统一下？

Comment (zhiqiu) [reply paddle/fluid/operators/bernoulli_op.cu]: done, thx

**Review by chenwhql — APPROVED** [review]
LGTM

**Review by jzhang533 — APPROVED** [review]
lgtm

**Pull request merged**